src/components/Person/Person.jsx

```diff
@@ -1 +1,27 @@
-// export const Person = ({ person }) => ();
+export const Person = (props) => {
+  const { name, age, sex, isMarried, partnerName } = props.person;
+
+  const partnerTitle = sex === 'f' ? 'husband' : 'wife';
```

Comment:
use CONSTANT for sex, for example `const FIMALE = 'f'`, try to avoid magic values